soundcard/pulseaudio.py

```diff
@@ -46,6 +46,43 @@ def func_with_lock(*args, **kwargs):
         self._pa_operation_unref(operation)
     return func_with_lock
 
+
+_channel_positions = {
+                         'left': _pa.PA_CHANNEL_POSITION_LEFT,
+                         'right': _pa.PA_CHANNEL_POSITION_RIGHT,
+                         'center': _pa.PA_CHANNEL_POSITION_CENTER,
+                         'subwoofer': _pa.PA_CHANNEL_POSITION_SUBWOOFER} | \
+                     {
+                         _ffi.string(_pa.pa_channel_position_to_string(idx)).decode('utf-8'): idx for idx in
+                         range(_pa.PA_CHANNEL_POSITION_MAX)
+                     }
+
+
+def get_channel_positions():
+    """
+    Return a dict containing the Pulseaudio channel position enum type index
+    for every channel position name string.
+    """
+    return _channel_positions
```

Comment:
This should probably be built on-the-fly here instead of a global.

Reply:
See my comment in https://github.com/bastibe/SoundCard/pull/169#discussion_r1123555175